src/modules/services/keycloak.nix

```diff
@@ -0,0 +1,498 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+
+let
+  cfg = config.services.keycloak;
+
+  inherit (lib)
+    mkIf
+    mkMerge
+    mkOption
+    mkRenamedOptionModule
+    mkPackageOption
+    types
+    ;
+
+  inherit (types)
+    nullOr
+    oneOf
+    listOf
+    attrsOf
+    ;
+
+  assertStringPath =
+    optionName: value:
+    if builtins.isPath value then
+      throw ''
+        services.keycloak.${optionName}:
+          ${builtins.toString value}
+          is a Nix path, but should be a string, since Nix
+          paths are copied into the world-readable Nix store.
+      ''
+    else
+      value;
+in
+{
+  imports = [
+    (mkRenamedOptionModule [ "keycloak" "enable" ] [ "services" "keycloak" "enable" ])
```

Comment:
Not sure I understand why this is needed, `keycloak.enable` did not exist before this PR?

Reply:
:+1: : Thanks thats a mistake from NixOS pkgs!